### ISSUE TITLE
packages get gives errors

### ISSUE THREAD

**Issue by mrbm** [jbarr21/flews#1]
Hello, trying to run your demo app but getting some road blocks:
```

$ flutter packages get
Waiting for another flutter command to release the startup lock...
Running "flutter packages get" in flews...                
Incompatible version constraints on mockito:
- flews depends on version 2.2.0
- flutter_test 0.0.0 depends on version 2.2.3
pub get failed (1)
```
So I will change the pubspec.yaml file to mockito: 2.2.3

The next error I get is:

```
$ flutter packages get
Running "flutter packages get" in flews...
Package built_value has no versions that match >=4.2.0 <5.0.0 derived from:
- flews depends on version ^4.2.0
pub get failed (1)
```
So going here: https://pub.dartlang.org/packages/built_value/versions

I see that there is a version 4.2.0

However this is where things start to not work....so I think people will get stuck just trying to get this demo running...

Thoughts?

**Comment by jbarr21** [comment]
@mrbm can you try using `./flutterw packages get`?

**Comment by realcarlos** [comment]
still failed ,some packages version need to be upgraded

**Comment by jbarr21** [comment]
resolved by ce92423
